include/eve/arch/as_register.hpp

```diff
@@ -10,9 +10,10 @@
 #include <eve/arch/cpu/as_register.hpp>
 
 #if !defined(EVE_NO_SIMD)
-#include <eve/arch/x86/as_register.hpp>
-#include <eve/arch/ppc/as_register.hpp>
-#include <eve/arch/arm/sve/as_register.hpp>
-#include <eve/arch/arm/neon/as_register.hpp>
+#  include <eve/arch/arm/neon/as_register.hpp>
```

Comment:
this is noise.

Reply:
Accidental change by clang-format. Fixed.